docs/scos/dev/the-docker-sdk/202204.0/configure-services.md

```diff
@@ -454,89 +447,138 @@ BLACKFIRE_CLIENT_ID={client_id} BLACKFIRE_CLIENT_TOKEN={client-token} docker/sdk
 It is not obligatory to pass all the details as environment variables or define all the details in the deploy file. You can pass the details in any combination.
 
+{% endinfo_block %}
+
 ## New Relic
 
-[New Relic](https://newrelic.com/) is a tool used to track the performance of services, environment to quickly find and fix issues.
+[New Relic](https://newrelic.com/) is a tool used to track the performance of services and the environment to quickly find and fix issues.
 
-The solution consists of a client and a server. The client is used to collect the data about applications in an environment and send it to the server for further analysis and presentation. The server is used to aggregate, analyse and present the data.
+The solution consists of a client and a server. The client is used to collect data about applications in an environment and send it to the server for further analysis and presentation. The server is used to aggregate, analyse, and present the data.
 
-### Configure New Relic
+### Prerequisites
 
-Follow the steps to enable New Relic:
+* Access to New Relic with an APM [account](#configure-new-relic-paas).
+* A New Relic license key.
+* The New Relic module.
 
-1. Adjust `deploy.*.yml` in the `docker:` section:
+Spryker provides its own New Relic licenses for use with its PaaS environments. A New Relic license key is only required if you wish to set up your own local monitoring.
 
-```yaml
-docker:
-    newrelic:
-        license: {new_relic_license}
+#### Installing the New Relic module
+
+While most environments may come with New Relic already available, you may need to include the module for your project. Available from Spryker, you can find the [Spryker new-relic module](https://github.com/spryker-eco/new-relic) within our module repositories. To include this repository in your project, add it to your `composer.json` file as follows:
+
+```bash
+composer require spryker-eco/new-relic
 ```
 
-2. Adjust `deploy.*.yml` in the `image:` section:
+### Configure New Relic (PaaS)
+
+To enable New Relic for your PaaS environment, in the `image` section, adjust `deploy.*.yml`:
 
 ```yaml
 image:
-    tag: spryker/php:7.4 # the image tag that has been previously used in `image:`
+    tag: spryker/php:7.4 # the image tag that has been previously used in `image`
     php:
         ...
         enabled-extensions:
             ...
             - newrelic
 ```
 
-#### Alternative configuration
+Once New Relic has been enabled in your deploy file, [contact Spryker Support](/docs/scos/user/intro-to-spryker/support/how-to-contact-spryker-support.html) to have it integrated into your environment. This can be done by submitting an infrastructure change request through the [Support Portal](/docs/scos/user/intro-to-spryker/support/how-to-use-the-support-portal.html). The Support team will confirm that a New Relic APM account is available for you and ensure that the correct application naming convention is set up to cascade to the appropriate APM.
```

Comment:
Do I need to push and deploy the updated file before contacting support?

Reply:
As I do not have access to push to any branches within spryker/spryker-docs, I have made this change to my initial PR #1331.

Reply:
I'm asking about the instructions in the doc, not the PR :D 

Meaning, do I need to push and deploy the updated deploy file before contacting support?

Reply:
I have further updated the language in my own PR at commit [9f6b7ebb777fe3f1ef0c0defb5311d3453946a56](https://github.com/spryker/spryker-docs/pull/1331/commits/4c4f3061940f0abba6984d670f6b77e0b671a576).